sieve/sieve-lex.l

```diff
@@ -382,10 +382,13 @@ ws              [ \t]+
 <INITIAL>jmapquery             return JMAPQUERY;
 
     /* vnd.cyrus.imip */
-<INITIAL>processimip          return PROCESSIMIP;
-<INITIAL>:invitesonly         return INVITESONLY;
+<INITIAL>processcalendar      return PROCESSIMIP;
+<INITIAL>processimip          return PROCESSIMIP;     /* legacy */
+<INITIAL>:allowpublic         return ALLOWPUBLIC;
+<INITIAL>:invitesonly         return INVITESONLY;     /* FM-specific */
 <INITIAL>:updatesonly         return UPDATESONLY;
-<INITIAL>:deletecanceled      return DELETECANCELED;
+<INITIAL>:deletecancelled     return DELETECANCELED;
```

Comment:
While the parameter is renamed from `:deletecanceled` to `:deletecancelled`, the token in sieve-lex.l is still with a single L.  This could lead to confusions during maintainance.